### ISSUE TITLE
Training on FB15k-237 without pre-trained embeddings. 

### ISSUE THREAD

**Issue by svjan5** [daiquocnguyen/ConvKB#6]
Hi,
I am unable to reproduce the results reported on `FB15k-237` without using the given pre-trained embeddings. Also, providing `--pre-trained` as `True/False` doesn't make any difference because in either case it is set to `True` ([reference](https://stackoverflow.com/questions/15008758/parsing-boolean-values-with-argparse)). The provided training command for `FB15k-237` is
``` shell
python train.py --embedding_dim 100 --num_filters 50 --learning_rate 0.000005 --name FB15k-237 --useConstantInit --model_name fb15k237
```
This uses a very small learning rate (almost 0), so basically it is not changing the provided pre-trained embeddings during training. Please provide the hyperparameters you used for learning the pre-trained embeddings.

Thanks in advance

**Comment by daiquocnguyen** [comment]
For your purpose, you just delete or comment Lines 58-76. The pre-trained embeddings was learned by TransE, and you can see the TransE hyper-parameters in our paper.
